sys-cluster/crmsh/crmsh-4.4.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{9,10} )
+
+if [[ ${PV} == *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/crmsh/crmsh"
+	inherit git-r3
+	S="${WORKDIR}/${PN}-${MY_TREE}"
+else
+	SRC_URI="https://github.com/crmsh/crmsh/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~hppa ~x86"
+fi
+
+inherit autotools python-single-r1
+
+DESCRIPTION="Pacemaker command line interface for management and configuration"
+HOMEPAGE="https://crmsh.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}
+	>=sys-cluster/pacemaker-2.1.2"
+RDEPEND="${DEPEND}
+	$(python_gen_cond_dep '
+		dev-python/lxml[${PYTHON_USEDEP}]
+		dev-python/parallax[${PYTHON_USEDEP}]
+		dev-python/pyyaml[${PYTHON_USEDEP}]
+	')
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-utils-wait4dc.diff"
```

Comment:
I am not quite sure if that is a problem, but the patch files in ::gentoo are usually named `.patch`, not `.diff`

Reply:
somewhere i read .patch or .diff. I used diff for comparison, so i named it diff

Reply:
You can simply rename the ending and edit this line in the ebuild, the file stays the same.